mantle/cmd/kola/kola.go

```diff
@@ -568,8 +568,13 @@ func syncFindParentImageOptions() error {
 		if err != nil {
 			return err
 		}
+	case "gce":
```

Comment:
should this be `gcp`?

Reply:
I think kola only knows `gce` right now. We should probably deprecate `gce` and move it to `gcp`.